src/main/java/com/yahoo/bullet/rest/model/WebSocketRequest.java

```diff
@@ -0,0 +1,21 @@
+/*
+ *  Copyright 2018, Yahoo Inc.
+ *  Licensed under the terms of the Apache License, Version 2.0.
+ *  See the LICENSE file associated with the project for terms.
+ */
+package com.yahoo.bullet.rest.model;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@NoArgsConstructor @Getter @Setter
+public class WebSocketRequest {
```

Comment:
Call this Type since it's nested inside WebSocketRequest anyway?

Reply:
Done.